dist/tools/openocd/openocd.sh

```diff
@@ -191,8 +191,16 @@ do_debug() {
     test_elffile
     test_ports
     test_tui
+    # setsid is needed so that Ctrl+C in GDB doesn't kill OpenOCD
+    [ -n "$(which setsid)" ] && SETSID=setsid
```

Comment:
The purpose was to allow overriding it from the environment in the case
where you have the osx workaround installed somewhere outside of the normal
path
On Aug 27, 2015 3:23 PM, "Daniel Krebs" <notifications@github.com> wrote:

> In dist/tools/openocd/openocd.sh
> <https://github.com/RIOT-OS/RIOT/pull/3619#discussion_r38093810>:
>
> > @@ -191,8 +191,16 @@ do_debug() {
> >      test_elffile
> >      test_ports
> >      test_tui
> > +    # setsid is needed so that Ctrl+C in GDB doesn't kill OpenOCD
> > +    [ -n "$(which setsid)" ] && SETSID=setsid
>
> Anyway, where is $SETSID supposed to be set?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/3619/files#r38093810>.
>